keen/explorer
# Fixes #112: Missing table view option

## What does this PR do? How does it affect users?
This ensures the proper data type is being set and that the table option is available when it should be.

## How should this be tested?

Try running a query with an interval and ensuring that the `table` view type is available to be used and works as expected.

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:


## Related tickets?
#112
